Utils/github_workflow_scripts/run_secrets_detection.py

```diff
@@ -41,6 +42,12 @@ def trigger_generic_webhook(options):
     # post to Content Gold
     res = requests.post(secrets_instance_url, json=body, auth=(username, password))
 
+    if res.status_code >= 500:
+        # Try again after two minutes in case getting 500 errors codes
+        # as there are some cases with small connection problems
+        sleep(120.0)
+        res = requests.post(secrets_instance_url, json=body, auth=(username, password))
```

Comment:
@israelpoli can you please do more than 1 retry? i would say to retry 3 times with sleep of 10 seconds between each http request